Shared/Services/LoopringGraphQLService.cs

```diff
@@ -884,7 +884,7 @@ public LoopringGraphQLService()
             //avoid query errors with search strings that cannot be converted to bytes
             //extra searchTermBytes is only filled if it matches strict RegEx, starting with 0x (added if missing)
             //and then any number of pairs "({2})+" of 0-9, a-f, A-F, end must be reached = $
-            string searchTermBytes = searchTerm.StartsWith("0x") ? searchTerm : "0x" + searchTerm;
+            string searchTermBytes = searchTerm.ToLower().StartsWith("0x") ? searchTerm.ToLower() : "0x" + searchTerm.ToLower();
```

Comment:
Personally, I'd find `string searchTermBytes = (searchTerm.StartsWith("0x", StringComparison.InvariantCultureIgnoreCase) ? searchTerm : "0x" + searchTerm).ToLower();` more to the point.